examples/aws-docker-build-deploy-dev.yaml

```diff
@@ -0,0 +1,75 @@
+name: <App> - Deployment [Dev]
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    name: Step1
+    uses: strongsdcom/github-actions/.github/workflows/aws-docker-build.yaml@main
+    secrets:
+      build-args: |
+        <App>_API_BASE_URL=https://<App>-dev.strongsd.com/api/         
```

Comment:
I believe this set of args depends on the application. Would be enough to use something like:
SOME_APP_ENV=xxx
SENTRY_RELEASE=<<short-sha>>